DEVELOPER_GUIDE.md

```diff
@@ -141,7 +141,7 @@ Run from `bundle-workflow` before making pull requests.
 cd bundle-workflow
 
 pipenv run isort .
-pipenv run black .
+git status -s | grep -e "[MA?]\s.*.py" | cut -c4- | xargs pipenv run black
```

Comment:
This is cryptic...

Reply:
Trust me ;) It just picks all .py files that are added or modified, minus the deleted ones.

Reply:
I opened https://github.com/opensearch-project/opensearch-build/issues/442 to add a pre-commit hook.